robustredirects/models.py

```diff
@@ -62,3 +63,11 @@ class Meta:
     def __unicode__(self):
         return _("Redirect: %(from)s --> %(to)s") % {'from': self.from_url, 'to': self.to_url}
 
+
+    @staticmethod
+    def ignored_url_paths():
```

Comment:
Interesting, why did you put this on the model instead of just making a function?

Reply:
A holdover from more OO-focused languages:  I like to keep things namespaced.

Reply:
Ah. Unlike Java Python's modules are namespaces.

```python
import my_utils

my_utils.my_function()
```

That would be the way to namespace this IMHO.

Little know fact, everything in Python is an object. Even the modules. https://docs.python.org/2/c-api/module.html